sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -510,7 +510,7 @@ public virtual async Task<Response> RemoveParticipantAsync(CommunicationUserIden
             scope.Start();
             try
             {
-                return await _chatThreadRestClient.RemoveChatParticipantAsync(Id, user.Id, cancellationToken).ConfigureAwait(false);
+                return await _chatThreadRestClient.RemoveChatParticipantAsync(Id, null, CommunicationIdentifierSerializer.Serialize(user).CommunicationUser, null, null, cancellationToken).ConfigureAwait(false);
```

Comment:
lets look at this together, want to understand how the rawId is handled in general ..